_posts/1-2-9-DropDownMenu.md

```diff
@@ -53,7 +53,6 @@ DropDownMenu is a full-screen contextual menu for displaying lists of items.
 
 ## Examples
 
-#### JSX declaration
```

Comment:
@tenodi why did you remove this label? we have it on many places (it should be there for each example of each component)

Check this commit for example? https://github.com/shoutem/shoutem.github.io/commit/2dd29cd5d79f7ec2f9d11cb1da200a50532c8bc5#diff-c67f66d7d4c914a0027f9bb259df08c4R66